### ISSUE TITLE
SMAP player rotation is bugged

### ISSUE THREAD

**Issue by theflyingzamboni** [Legend-of-Dragoon-Modding/Severed-Chains#981]
I noticed a couple things, one definite, one a little less certain.

Definite: The rotation only applies in one direction, unlike retail. If you run back and forth with Dart horizontally, he will rotate when turning around to go left, but reverse direction immediately when going right.

Less certain: The way Dart rotates if you face left and tap down seems to me slightly jerkier in SC than retail.

**Comment by LordMonoxide** [comment]
Not getting the left-right thing now but running frame by frame you can clearly see that there is no interpolation being applied to his rotation. He only rotates every other frame.

**Comment by LordMonoxide** [comment]
I was able to cause the left -> right snap by running around in circles ccw a few times

**Comment by LordMonoxide** [comment]
Yup, rotating cw will make your rotation go positive and ccw will make it go negative when crossing 0

**Comment by LordMonoxide** [comment]
You can keep winding your angle up to a certain point too, I'm up to A Y rotation of 11
